src/discordas.ts

```diff
@@ -94,6 +94,13 @@ async function run(port: number, fileConfig: DiscordBridgeConfig) {
                 } catch (err) { log.error("Exception thrown while handling \"onAliasQuery\" event", err); }
             },
             onEvent: async (request) => {
+                // tslint:disable-next-line no-any
+                const done = function(resolve: (res: any) => void, reject: (err: Error) => void) {
+                    // tslint:disable-next-line no-invalid-this no-floating-promises
+                    (this as Promise<{}>).then(resolve);
+                    // tslint:disable-next-line no-invalid-this no-floating-promises
+                    (this as Promise<{}>).then(reject);
```

Comment:
Can you explain this? This looks like we call both the resolve and reject callback, thus resolving and rejecting the promise at the same time?

Reply:
That should be a catch